src/fsharp/FSharp.Compiler.Private/AssemblyInfo.fs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
+
+namespace Microsoft.FSharp
+
+open System.Reflection
+open System.Runtime.InteropServices
+
+#if STRONG_NAME_FSHARP_COMPILER_WITH_TEST_KEY
+[<assembly:System.Runtime.CompilerServices.InternalsVisibleTo("fsc, PublicKey=002400000480000094000000060200000024000052534131000400000100010077d32e043d184cf8cebf177201ec6fad091581a3a639a0534f1c4ebb3ab847a6b6636990224a04cf4bd1aec51ecec44cf0c8922eb5bb2ee65ec3fb9baa87e141042c96ce414f98af33508c7e24dab5b068aa802f6693881537ee0efcb5d3f1c9aaf8215ac42e92ba9a5a02574d6890d07464cb2f338b043b1c4ffe98efe069ee")>]
```

Comment:
These should be handled by the `<InternalsVisibleTo />` elements in the project file.

Reply:
@brettfo That stuff doesn't work in the Mono build.  Or perhaps it's using the wrong key?

```
      <_PublicKey>002400000480000094000000060200000024000052534131000400000100010007D1FA57C4AED9F0A32E84AA0FAEFD0DE9E8FD6AEC8F87FB03766C834C99921EB23BE79AD9D5DCC1DD9AD236132102900B723CF980957FC4E177108FC607774F29E8320E92EA05ECE4E821C0A5EFE8F1645C4C0C93C1AB99285D622CAA652C1DFAD63D745D6F2DE5F17E5EAF0FC4963D261C8A12436518206DC093344D5AD293</_PublicKey>

```

Reply:
The key can be overridden via [this](https://github.com/Microsoft/visualfsharp/blob/5728b68d9fea67aaf45f810b2f602419c1094b92/src/FSharpSource.targets#L388) like so:

``` xml
<PropertyGroup Condition="not mono">
  <InternalsVisibleTo Include="fsc" /><!-- will use the hard-coded key -->
</PropertyGroup>
<PropertyGroup Condition="mono">
  <InternalsVisibleTo Include="fsc" Key="1234" />
</PropertyGroup>
```

Reply:
Ah ok, will try that, thanks